src/utils.ts

```diff
@@ -21,13 +22,28 @@ export async function encodePayload(payload: Record<string, any>): Promise<Encod
   }
 }
 
-export function encodeCleartext(cleartext: Record<string, any>): CID {
+export function pad(b: Uint8Array, blockSize = ENC_BLOCK_SIZE): Uint8Array {
+  // pads with 1 followed by 0s
+  const padLen = blockSize - (b.length % blockSize)
+  return u8a.concat([b, new Uint8Array([128]), new Uint8Array(padLen - 1)])
```

Comment:
Not an issue but maybe `128` could be set as a constant to avoid having "magic numbers" in functions?